introduction.md

```diff
@@ -0,0 +1,2 @@
+Hello, my name is Ben, a recent college graduate who would like to master Github, which is why I'm taking this Code School course. 
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.